test/test_Simulation.jl

```diff
@@ -31,17 +31,25 @@ end # testset
 end
 
 
+world2 = World(
+    v0=1.0,
+    n_dogs=100,
+    boxsize=50.0,
+    motion=BrownianMotion(noise=0.5, friction=1.0),
+    sheeps=DenseSheeps(20, n_sheeps=20),
+)
+
```

Comment:
generally speaking it is not a good idea to have more collectors than items, since you might end up in the situation where all items are picked up and in the best case the simulation will just get stuck in that state